modules/scanner-requirements.adoc

```diff
@@ -0,0 +1,24 @@
+// Module included in the following assemblies:
+//
+// * installing/prerequisites.adoc
+:_module-type: CONCEPT
+[id="scanner-prerequisites_{context}"]
+= Prerequisites for installing Scanner
+
+{product-title} includes an image vulnerability scanner called _Scanner_.
```

Comment:
This is the only place in these prereq docs where you italicize the component name (Scanner). Maybe not needed?